ex03.ex

```diff
@@ -55,7 +55,10 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+
+  def odd_even([ head | tail ]) when Integer.is_even(head), do: [ :even | odd_even tail ]
```

Comment:
Thank you for using guards on both function heads: not doing so would be an error.

A minor stylistic point: would

~~~ elixir
def odd_even([ head | tail ]) when Integer.is_even(head), do: [ :even | odd_even tail ]
~~~

be clearer with parens?

~~~ elixir
def odd_even([ head | tail ]) when Integer.is_even(head), do: [ :even | odd_even(tail) ]
~~~